draft-ietf-tls-tls13.md

```diff
@@ -3400,7 +3400,7 @@ the forward secrecy of data sent before the sender changed keys.
 
 If implementations independently send their own KeyUpdates with
 request_update set to "update_requested", and they cross in flight, then each side
-will also send a response, with the result that each side increments
+MUST also send a response, with the result that each side increments
```

Comment:
I think we can remove the MUST here b/c the MUST requirement is above. This is telling you how things behave.